spec/requests/rooms_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Room viewing', type: :request do
+  it 'displays current Jam information if it exists' do
+    # Lock datetime to Jan 24, 2020 1:4:44
+    travel_to Time.zone.local(2020, 1, 24, 0o1, 0o4, 44)
```

Comment:
`travel_to` takes a block since Rails 5.2 so it's essentially `Timecop.freeze`, no `travel_back` required